eng/Versions.props

```diff
@@ -111,7 +111,6 @@
     <MicrosoftNETFrameworkReferenceAssembliesnet20Version>1.0.0</MicrosoftNETFrameworkReferenceAssembliesnet20Version>
     <jnm2ReferenceAssembliesnet35Version>1.0.1</jnm2ReferenceAssembliesnet35Version>
     <MicrosoftNETCoreTestHostVersion>1.1.0</MicrosoftNETCoreTestHostVersion>
-    <MicrosoftNetFX20Version>1.0.3</MicrosoftNetFX20Version>
```

Comment:
Can we also delete src\Dependencies\Microsoft.NetFX20?

Reply:
@tmat do we need this package anymore? Happy to delete if it's not needed.